torch/csrc/jit/mobile/module.cpp

```diff
@@ -129,6 +129,11 @@ const std::map<std::string, at::Tensor> Module::named_parameters() const {
   slot_named_params_recurse(object_, &params, name);
   return params;
 }
+
+std::string Module::get_forward_method_debug_info(size_t pc) {
```

Comment:
This method seems to be only used in tests and doesn't look general enough to be in the Module API. Would it make sense to convert it to a helper function in the test?

Reply:
I think we do need it to display module info when performing latency profiling. @kimishpatel

Reply:
@ZolotukhinM, so the idea right now is this: For each PC, there is a corresponding debug info available that the module allows you to query wrt its forward method. So then in lite predictor that uses loads `mobile::Module` and runs the forward method of it, can query the current PC and get corresponding debug info.
This is honestly not super clean, but the existing operator profiling infra was built this way. https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/mobile/interpreter.cpp#L43. Definitely not ideal, but I think we will have to fix that in longer term. I would prefer not having Tai fix issues we introduced ourselves :)

Reply:
On the other hand, I think we can do something like this. We can ask lite interpreter to give debug info of current pc, so like `get_current_pc_debug_info()`, that returns you the string.
Then in your test code, as well as in lite predictor, as we run the model, we can have onFunctionEnter/Exit calls that get called as if when we are doing profiling. And within this call you can collect all the debug info and check against what you expect. And you do this only once.
```
in OnFunctionEnter
if (debug_info_wanted) {
debug_info = module.get_current_pc_debug_info()
append debug info to vector.
Only in first forward method call debug info is extracted and for later runs
get_debug_info = false.
}
```